applications/advanced-concepts/bridging.md

```diff
@@ -0,0 +1,16 @@
+---
+parent: Building and research
+title: Bridging
+description: Two-step translation through an intermediate language
+---
+
+**Bridging** uses an intermediate language to translate between source and target languages.
```

Comment:
Thank you for your PR! :)

I'd make the intro sentence a little bit more like a definition, to keep consistency with the rest of the articles.

Something like:
---

**Bridging** is a machine translation method that uses an intermediate language to translate between source and target. languages.

---

Reply:
I wonder if hyperlinking the `languages` section is always necessary.  The thing is that most (if not all) articles use the term `language`. Do you think it is a good idea to link it every time or perhaps only in some articles?

Reply:
Only some, for instance if there is significant information in the language-specific pages that we're trying to link to. For instance, if we mention language families that might make sense to link. Or if we're trying to offer a pointer to figure out the most common scripts for each language.